### ISSUE TITLE
Website Accessibility-WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (a)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14494]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
li:nth-child(7) > a
#### Snippet
```<a href="http://stackoverflow.com/questions/tagged/bazel">
                <span class="visible-xs">StackOverflow</span>
                <span><i class="nav-icon fa fa-stack-overflow hidden-xs"></i></span>```


````

How to fix | 
Fix the following:
Element is in tab order and does not have accessible text

Fix ONE of the following:
Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelled by attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

Environment
Microsoft Edge version 96.0.1054.62

**Comment by fweikert** [comment]
We've migrated to a new site at https://bazel.build that fixes several accessibility issues.
